src/instruments/src/EFB/Dashboard/Widgets/FlightWidget.tsx

```diff
@@ -1,58 +1,87 @@
-                        href="/dashboard/secondary"
-                        style={props.name !== "todays" ? {borderBottom: "solid #00C2CB .1em"} : {borderBottom: "none"}}
-                    >
-                        Secondary
-                    </a>
+        <div className="bg-gray-800 rounded-xl p-6 text-white shadow-lg text-base">
+            <div id={'flight-' + props.name}>
+                <div className="text-center mb-6">
+                    <h1 className="text-xl leading-5">{props.airline + props.flightNum}</h1>
+                    <span className="text-sm">{props.aircraftReg}</span>
                 </div>
-                <div id='origin-destination'>
-                    <span>{props.dep}</span>
-                    <i className="material-icons">send</i>
-                    <span>{props.arr}</span>
+
+                <div className="flex items-center justify-center mb-6 text-base">
+                    [BOM] <span className="mx-3 text-3xl">{props.dep}</span>
+                    <IconPlane size={35} stroke={1.5} strokeLinejoin="miter" />
+                    <span className="mx-3 text-3xl">{props.arr}</span> [DEL]
                 </div>
 
-                <div id="Time">
-                    <p className="Title">TIME</p>
-                    <p>{props.name === "todays" ? props.timeSinceStart : "01:43"}</p>
+                <div className="flex mb-6">
+                    <div className="w-1/2 mr-4">
+                        <div className="flex justify-end">
+                            STD <IconPlaneDeparture className="ml-2" size={23} stroke={1.5} strokeLinejoin="miter" />
+                        </div>
+                        <div className="text-right mt-1">{props.std}z</div>
+                    </div>
+                    <div className="w-1/2 ml-4">
+                        <div className="flex justify-start">
+                            <IconPlaneArrival className="mr-2" size={23} stroke={1.5} strokeLinejoin="miter" /> STA
+                        </div>
+                        <div className="text-left mt-1">{props.sta}z</div>
+                    </div>
                 </div>
 
-                <div id="Distance">
-                    <p className="Title">DISTANCE</p>
-                    <p>{props.distance}</p>
+                <div className="flex justify-center text-center font-mono mb-6 max-w-md text-sm">
+                    AKRIB Q23 INTIL W19 IKAVA Q23 IBANI W19 BPL W20 HIA W71 LURGI Q21 TELUV [{props.distance}]
                 </div>
 
-                <div id="ETA">
-                    <p className="Title">{props.name === "todays" ? "ETA (UTC)" : "ETA (UTC)"}</p>
-                    <p>{props.eta}</p>
+                <div className="text-sm">
+                    <button onClick={() => props.fetchSimbrief()} className="w-full bg-blue-500 p-2 text-white flex items-center justify-center rounded-lg mb-2 focus:outline-none">
+                        FRM SIMBRIEF
```

Comment:
Why abbreviate this? Same with LNK MCDU
There is plenty of room to add one more letter

Reply:
Just a placeholder for now, I'll change this in final commits :)